internal/modules/overview/overview.go

```diff
@@ -259,38 +259,6 @@ func (co *Overview) Content(ctx context.Context, contentPath string, opts module
 
 func (co *Overview) extensionDescriber(path, namespace string, options describer.Options) (*component.Extension, error) {
```

Comment:
`extensionDescriber` can also be entirely removed along with the generatorFunc in `generator.go.